packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -472,7 +472,13 @@ abstract class BaseMethod implements PythonBase {
             paramNames.push(toPythonParameterName(param.name));
         }
```

Comment:
Wouldn't it make more sense to simply check if `param.variadic` is true here and push a different value to `paramNames`?

Reply:
Yeah. I started with a change proposed in the original issue. Meant to clean it up but forgot.  Testing my updates now.